Source/Core/DolphinWX/Config/AudioConfigPane.cpp

```diff
@@ -55,50 +56,54 @@ void AudioConfigPane::InitializeGUI()
   m_dpl2_decoder_checkbox->SetToolTip(
       _("Enables Dolby Pro Logic II emulation using 5.1 surround. Certain backends only."));
 
-  const int space5 = FromDIP(5);
+  m_soundtouch_timestretch_checkbox->SetToolTip(_("Enables timestretching support, smoother audio "
```

Comment:
Maybe something like "Adjusts audio pitch when not running at full speed (adds latency)"?